src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,147 @@
+package duke.command;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.Scanner;
+
+
+public class Duke {
+    public static void listOut(Task[] tasks, int taskNumber) {
```

Comment:
Consider using a clearer naming convention for your method such as printTasks().